stress-ng.h

```diff
@@ -844,6 +844,7 @@ typedef unsigned long int __kernel_ulong_t;
 #define CLASS_PATHOLOGICAL	(0x00002000)	/* can hang a machine */
 
 
+
```

Comment:
I'd prefer this empty line to not be added.

Reply:
Sure, will remove in v2